mtp_noms_ops/assets-src/stylesheets/views/_checks.scss

```diff
@@ -165,3 +167,51 @@ table.mtp-table--review {
 .mtp-check-header__assigned-user {
   color: $red;
 }
+
+.mtp-check-cell__added-to-my-list::before {
+	content: "✓";
```

Comment:
ooh this is interesting. so this creates an inline element before a td, if i read it correctly? or does it actually go inside the td but before the rest of the td's contents?
i only ask cos if it's outside the td, then it's between two table cells and it might not render consistently in different browsers… maybe?

Reply:
As it's a pseudo element, it's not really in the DOM at all, although when rendered as if it is, it's still "inside" the element it's applied to, so that's OK I think